pkg/vulnsrc/vulnerability/vulnerability.go

```diff
@@ -176,6 +179,19 @@ func getReferences(details map[string]types.VulnerabilityDetail) []string {
 	return refs
 }
 
+func getRejectedStatus(details map[string]types.VulnerabilityDetail) bool {
+	for _, source := range sources {
+		d, ok := details[source]
+		if !ok {
+			continue
+		}
+		if strings.HasPrefix(d.Description, rejectVulnerability) {
```

Comment:
`strings.Contains` may be sufficient. Could you check if `strings.Contains` has a problem? I mean we have to make sure that there is no vulnerability which includes `** REJECT **` in the description, but is not rejected vulnerability.
If we can use `strings.Contains`, we may not need the process adding `** REJECT **` prefix in each file like `nvd.go` and `redhat.go`.

Reply:
@knqyf263 I prefixed the rejected because in case of redhat we take the first not empty description as vulnerability description, and there can be the case that 2nd or 3rd description specifies the infor for rejection
if we are 100% sure that it won't break, then we can use strings.Contains
https://github.com/aquasecurity/trivy-db/blob/master/pkg/vulnsrc/nvd/nvd.go#L80

Reply:
Yes, I know. This is the reason I said as follows.
>Could you check if strings.Contains has a problem? I mean we have to make sure that there is no vulnerability which includes ** REJECT ** in the description, but is not rejected vulnerability.